rpm_spec/subpackages/manageiq-core

```diff
@@ -43,6 +43,8 @@ done
 #  so root and manageiq users can read them.
 [ -e %{app_root}/certs/v2_key ] && %{__chown} manageiq.manageiq %{app_root}/certs/v2_key
 [ -e %{app_root}/certs/v2_key ] && %{__chmod} o-rw %{app_root}/certs/v2_key
+[ -e %{app_root}/certs/server.cer ] && %{__chmod} g+rw %{app_root}/certs/server.cer
```

Comment:
group needs write permission as well, or just read?

Reply:
discussion about this here -> https://github.com/ManageIQ/manageiq-appliance/pull/365#discussion_r949534675